dashboard/app/main.go

```diff
@@ -839,15 +842,43 @@ func loadCrashesForBug(c context.Context, bug *Bug) ([]*uiCrash, []byte, error)
 	return results, sampleReport, nil
 }
 
+func loadFixBisectionsForBug(c context.Context, bug *Bug) ([]*uiCrash, error) {
+	bugKey := bug.key(c)
+	jobs, _, err := queryJobsForBug(c, bugKey, JobBisectFix)
+	if err != nil {
+		return nil, err
+	}
+	var results []*uiCrash
+	for _, job := range jobs {
+		crash, err := queryCrashForJob(c, job, bugKey)
+		if err != nil {
+			return nil, err
+		}
+		if crash == nil {
+			continue
+		}
+		build, err := loadBuild(c, bug.Namespace, job.BuildID)
+		if err != nil {
+			return nil, err
+		}
+		results = append(results, makeUICrash(crash, build))
+	}
+	return results, nil
+}
+
 func makeUICrash(crash *Crash, build *Build) *uiCrash {
 	ui := &uiCrash{
-		Manager:      crash.Manager,
-		Time:         crash.Time,
-		Maintainers:  strings.Join(crash.Maintainers, ", "),
-		LogLink:      textLink(textCrashLog, crash.Log),
-		ReportLink:   textLink(textCrashReport, crash.Report),
-		ReproSyzLink: textLink(textReproSyz, crash.ReproSyz),
-		ReproCLink:   textLink(textReproC, crash.ReproC),
+		Manager:        crash.Manager,
+		Time:           crash.Time,
+		Maintainers:    strings.Join(crash.Maintainers, ", "),
+		LogLink:        textLink(textCrashLog, crash.Log),
+		ReportLink:     textLink(textCrashReport, crash.Report),
+		ReproSyzLink:   textLink(textReproSyz, crash.ReproSyz),
+		ReproCLink:     textLink(textReproC, crash.ReproC),
+		HasMaintainers: false,
+	}
+	if ui.Maintainers != "" {
```

Comment:
A shorter way to do this would be:
```
HasMaintainers:  ui.Maintainers != "",
```
But if it's just a check for empty string, then we don't need it at all. `if` predicate on a string in the templates will do check for emptiness, no need for a separate bool.